api/api.go

```diff
@@ -84,9 +85,41 @@ func (app *api) companyHandler(w http.ResponseWriter, r *http.Request) {
 		messageResponse(w, http.StatusNotFound, fmt.Sprintf("CNPJ %s não encontrado.", cnpj.Mask(v)))
 		return
 	}
+
+	//check if the url contains url param "fields"
+	command := r.URL.Query().Get("fields") // "" = returns all data.
+	if command == "" {
+		w.Header().Set("Content-type", "application/json")
+		w.WriteHeader(http.StatusOK)
+		io.WriteString(w, s)
+		return
+	}
+
+	//create a map to store the json
+	var data map[string]interface{}
+	json.Unmarshal([]byte(s), &data)
+
+	//if command contains ",", it means that the user wants multiple fields
+	if strings.Contains(command, ",") {
```

Comment:
Me pergunto se `if` é necessário? Tenho a impressão que se o `command` for `razao_social`, `strings.Split(command)` funcionaria igual: https://go.dev/play/p/Yqn_OiOxgM_t

Reply:
Tem razão. Vou remover este if.